test/src/main/java/org/apache/accumulo/test/functional/FunctionalTestUtils.java

```diff
@@ -218,4 +219,30 @@ private static FateStatus getFateStatus(ClientContext context, AccumuloCluster c
       throw new RuntimeException(e);
     }
   }
+
+  static long checkFlushId(AccumuloClient c, String tableName, long prevFlushID) throws Exception {
+    try (Scanner scanner = c.createScanner(MetadataTable.NAME, Authorizations.EMPTY)) {
```

Comment:
Could use ample for this instead of a scanner.  Not sure if the following compiles or works, just typed it up as a quick example.

```java
try(TabletsMetadata tablets = ((ClientContext)c).getAmple().readTablets(tableId).fetch(FLUSH_ID).build())){
      for(TabletMetadata tablet : tablets) {
          tablets.getFlushId();
       }      


}
```